.github/workflows/pr-labeler.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Label PRs with labels such as size. Note that this workflow is designed not to
+# fail if labeling actions encounter errors; instead, it writes error messages
+# as annotations on the workflow's run summary page. If labels don't seem to be
+# getting applied, check the run summary page for errors.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+name: Pull request labeler
+run-name: >-
+  Label pull request ${{github.event.pull_request.number}} by ${{github.actor}}
+
+on:
+  pull_request:
```

Comment:
This should be `pull_request_target` as in Cirq.  We tried with `pull_request` and it failed on permissions.  `pull_request` is also more dangerous, because it runs the workflow with the PR code.  `pull_request_target` runs the code in `master` which was code-reviewed.